### DISCUSSION TITLE
Lut calibration

### DISCUSSION THREAD

**Discussion by ShockWaveFlash** [awawa-dev/HyperHDR#516] [Q&A] [answered]
Hey all

I tried to do a lut calibration.

Unfortunately, the calibration always aborts with the indication that the resolution is wrong.

I use a 1920x1080 resolution.

with 1280x720 it does not work either.

do you have another idea?

**Comment by awawa-dev** [comment, marked as answer]
Hi Are you sure that resolution is working for your grabber .. overview/first tab confirms it? Post your HyperHDR full logs when that happens.
